src/index.js

```diff
@@ -1,5 +1,4 @@
 import 'tachyons-egghead'
-import 'font-awesome/css/font-awesome.min.css'
```

Comment:
No need for this anymore, since latest egghead-ui is now using SVGs so `import {Icon} from 'egghead-ui'` contains everything you need; peace out globals!